test/handler.test.js

```diff
@@ -195,6 +195,24 @@ describe('strong-error-handler', function() {
       });
     });
 
+    it('contains thrown message when debug=true', function(done) {
```

Comment:
I think this is a flaw that should be fixed in the unit-test above. Instead of using the same `for (var key in error)` loop to build the list of the expected properties, we should explicitly enumerate properties we are expecting to see. That would prevent this situation in the future, and also verify that `err.name` is set in the response.

```js
var error = new ErrorWithProps('a test error message', {
  details: ...,
  ...
});
// ...

var expectedData = {
  statusCode: 500,
  message: 'a test error message',
  name: 'ErrorWithProps',
  details: 'some details',
  extra: 'sensitive data',
  stack: error.stack,
});
```

After applying the change described above, this new test should not be necessary at all.

Thoughts?